pkg/git

```diff
@@ -30,6 +31,6 @@ dest="$butch_install_dir""$butch_prefix"
 rm -f "$dest"/lib/perl5/core_perl/perllocal.pod
 mkdir -p "$dest"/bin
 cp gitk-git/gitk "$dest"/bin/
-cp perl/private-Error.pm "$dest"/lib/perl5/site_perl/Error.pm
+install -Dm664 perl/FromCPAN/Error.pm "$dest"/lib/perl5/site_perl/Error.pm
```

Comment:
does that install command with -Dm664 in a single term work ?
well maybe even if it does, add the space just so its consistent with other packages

Reply:
it seems like there are a few different styles. ca-certificates uses the one I used, others do `install -D -m 664`, others do `install -Dm 644` and some use a few other variations

Reply:
well, let's not add yet another style